src/tools/crossgen2/ILCompiler.TypeSystem.ReadyToRun/ILCompiler.TypeSystem.ReadyToRun.csproj

```diff
@@ -7,12 +7,12 @@
     <TargetFramework>netstandard1.3</TargetFramework>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Platforms>x64;x86</Platforms>
+    <PlatformTargets Condition="'$(BuildArch)' != ''">$(BuildArch)</PlatformTargets>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 
     <!-- We're binplacing these into an existing publish layout so that F5 build in VS updates
          the same bits tests expect to see in bin/crossgen2. That way we never need to wonder which
          binaries are up to date and which are stale. -->
-    <OutputPath>$(BinDir)/crossgen2</OutputPath>
```

Comment:
Can we leave the conditioned OutputPath in the other projects (like in crossgen2.csproj)?

Reply:
There is no need for those. Building the main crossgen2 project takes care of putting these dependent files in the correct place